.github/ISSUE_TEMPLATE/5.priority-support.yml

```diff
@@ -0,0 +1,39 @@
+name: 'Priority Support: SLA ⏰'
```

Comment:
I wonder if this should be here (same with Base UI). Do we provide priority support for X subscribers?

Reply:
I was wondering the same, there were few other X related things that I copied over, cc @oliviertassinari?

Reply:
https://github.com/mui/material-ui/pull/37671

Reply:
We sold support on Core for a few customers, I believe, a handful. From their perspective, they need as much Material UI as the advanced components that extends Material UI, they compare us with Kendo UI or Syncfusion that offers the same product coverage. 

On the implementation, I believe the plan is to change this issue template to be broader, allow anyone to validate their support key and see what they have access to.